src/us/kbase/common/utils/CondorUtils.java

```diff
@@ -118,7 +114,7 @@ public static String clientGroupsToRequirements(String clientGroups) {
     /**
      * Remove space, remove single quote, and remove double quotes.
      * @param input
-     * @return
+     * @return cleaned string
```

Comment:
What's this?

Reply:
Fixed the comment. 
Condor doesn't like quotes in submission, so this protects against nasty clientgroups

Reply:
Ah, ok.  I wasn't paying enough attention and didn't realize this was in a comment.